scantool/diag_cfg.c

```diff
@@ -125,7 +125,7 @@ int diag_cfg_setopt(struct cfgi *cfgp, int optid) {
 //for u8 / int types, sprintf with %X and %d formatters respectively
 char *diag_cfg_getstr(struct cfgi *cfgp) {
 	char *str;
-	size_t len;
+	size_t len = 0;
```

Comment:
This popped only up in the bcc32x/bcc64 compiler configuration - not sure why the other configuration do not show this.